src/ophys_etl/utils/traces.py

```diff
@@ -1,11 +1,28 @@
 from typing import List
-from functools import partial
 import numpy as np
 from scipy.sparse import coo_matrix
 from scipy.ndimage.filters import median_filter
 
-# Partial for simplifying repeat median filter calls
-medfilt = partial(median_filter, mode='constant')
+
+def nanmedian_filter(x, filter_length):
```

Comment:
Is there not an equivalent setting of `scipy.ndimage.filters.median_filter` that will give the same results as the function defined below? No need to roll our own if we can reuse a standard library.

Reply:
I agree that would be ideal. But I don't see an existing function/arg in a package that allows for this from google/chatgpt.

Reply:
Right, the filter has to be nan median. Missed that when writing the comment.